config_defaults_inc.php

```diff
@@ -1955,6 +1955,28 @@
  */
 $g_file_upload_method = DATABASE;
 
+/**
+ * Zip_attachments in order to optimize  
+ * network performance and save disk/db space.
+ * @global integer $g_zip_attachments
+ */
+$g_zip_attachments		=	OFF;
+
+/**
+ * Zip_excludes defines the file types to omit from the zipping process 
+ * because the files are already compressed and/or one does not want to
+ * compress a specific file type.
+ * Values should be provided in lowercase.
+ * @global integer $g_zip_excludes
+ */
+$g_zip_attachments_excluded_extensions		= array('pdf','zip','docx','xlsx','pptx');
```

Comment:
What about dropping `$g_zip_attachments` and have just option `$g_zip_attachments_included_extensions` without any predefined extensions as default?
This would allow a more precise definition of what's wanted.
If we should stay on the blacklist approach, there should be some more extensions added (investigation needed) , e. g. 7z, jpg, jpeg, mov, mpeg, .... one more reason why I prefer the whitelist approach.

Reply:
I could change it to whitelisting. Before making any changes we should have consensus on the way forward.